RecoTracker/LSTCore/standalone/README.md

```diff
@@ -1,53 +1,100 @@
-# TrackLooper
+# How to set up standalone LST
 
-
-## Quick Start
-
-
-### Setting up LSTPerformanceWeb (only for lnx7188 and lnx4555)
+## Setting up LSTPerformanceWeb (only for lnx7188 and lnx4555)
 
 For lnx7188 and lnx4555 this needs to be done once
 
     cd /cdat/tem/${USER}/
     git clone git@github.com:SegmentLinking/LSTPerformanceWeb.git
 
-### Setting up container (only for lnx7188)
+## Setting up container (only for lnx7188)
 
 For lnx7188 this needs to be done before compiling or running the code:
 
     singularity shell --nv --bind /mnt/data1:/data --bind /data2/segmentlinking/ --bind /opt --bind /nfs --bind /mnt --bind /usr/local/cuda/bin/ --bind /cvmfs  /cvmfs/unpacked.cern.ch/registry.hub.docker.com/cmssw/el8:x86_64
 
-### Setting up the code
+## Setting up LST
+
+There are two way to set up LST as a standalone, either by setting up a full CMSSW area, which provides a unified setup for standalone and CMSSW tests, or by `sparse-checkout` only the relevant package and using them independent of CMSSW. A CVMFS-less setup is also provided for the second option.
+
+### Setting up LST within CMSSW (preferred option)
+
+```bash
+CMSSW_VERSION=CMSSW_14_2_0_pre4 # Change with latest/preferred CMSSW version
+LST_BRANCH=CMSSW_14_1_0_pre3_LST_X_LSTCore_realfiles_batch1_devel # Change to the appropriate branch
+cmsrel ${CMSSW_VERSION}
+cd ${CMSSW_VERSION}/src/
+cmsenv
+git cms-init
+git remote add SegLink git@github.com:SegmentLinking/cmssw.git
+git fetch SegLink ${LST_BRANCH}
+git checkout ${LST_BRANCH}
```

Comment:
Now that the LST PR is in CMSSW, these steps can in principle be skipped. Maybe add a comment that this is only needed to get some bleeding-edge development from some branch?

Reply:
Thank you, Manos! I addressed all these comments.